data/json/monsters/reptile_amphibian.json

```
@@ -600,17 +540,20 @@
     "id": "mon_strange_frog",
     "type": "MONSTER",
     "copy-from": "mon_bullfrog_frog",
-    "description": "Based on its vestigial tail, you might call it a polliwog.  That is if it weren't so large and didn't already move like an adult frog.  In any case, it looks to have some growing left to do.",
+    "description": "Based on its vestigial tail, you would surmise that this bullfrog still has some growing to do.  That is if it weren't already larger than any adult frog.",
     "name": { "str": "strange bullfrog" },
+    "volume": "4 L",
+    "weight": "3 kg",
+    "hp": 12,
     "harvest": "mutant_tiny",
-    "upgrades": { "age_grow": 120, "into": "mon_frog_small" },
+    "upgrades": { "age_grow": 15, "into": "mon_strange_bigfrog" },
     "reproduction": { "baby_egg": "egg_strange_frog", "baby_count": 16, "baby_timer": 26 }
   },
   {
     "id": "mon_fowler_toad",
     "type": "MONSTER",
     "copy-from": "mon_proxy_frog",
-    "description": "Fowler's toads secrete a harmful and distasteful compound from their backs.  This helps to safeguard them from predators and also gives them a distinctive odour like boiled peanuts.",
+    "description": "Fowler's toads secrete a harmful and distasteful compound from their backs.  This helps safeguard them from predators and also gives them a distinctive odour like boiled peanuts.",
```

```suggestion
    "description": "Fowler's toads secrete a harmful and distasteful compound from their backs.  This helps safeguard them from predators and also gives them a distinctive odor like boiled peanuts.",
```
Game set in America gets American English spellings